components/automate-ui/src/app/pages/team-details/team-details.component.ts

```diff
@@ -182,23 +199,34 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
     }));
   }
 
-  updateTeam(): void {
-    const newTeam = <Team>Object.assign(
-      {}, this.team, { name: this.editForm.get('teamName').value });
-    this.store.dispatch(new UpdateTeam(newTeam));
+  public keyPressed() {
+    this.saveSuccessful = false;
   }
 
-  setEditMode(isEditMode: boolean): void {
-    this.editForm.patchValue(
-      {
-        teamName: this.team.name
+  public saveNameChange(): void {
+    this.saveSuccessful = false;
+    this.saving = true;
+    const name: string = this.updateNameForm.controls['name'].value.trim();
```

Comment:
Suggest use non-strings where equivalent:
```suggestion
    const name: string = this.updateNameForm.controls.name.value.trim();
```